home_audit.cli.js

```diff
@@ -6,10 +6,11 @@
  * node home_audit.cli.js '{"solar_electric_capacity_known" : "3"}'
  * > {"solar_electric_capacity_known":"3 is outside the allowed range (0 - 1)"}
  */
+const fs = require('fs');
+
 let hes_validation_engine = require('./home_audit.node.js');
 let validate_address = hes_validation_engine.validate_address;
 let validate_home_audit = hes_validation_engine.validate_home_audit;
-
-const obj = JSON.parse(process.argv[2]);
+const obj = JSON.parse(fs.readFileSync(process.argv[2]));
```

Comment:
This is a breaking change for our API code, which passes the JSON directly.  I don't think having it accept a file is a bad idea, but we should continue to allow passing the string (which is also what's documented in the comments above).

Maybe...
```javascript
let input = process.argv[2];
if(fs.existsSync(input)) {
    input = fs.readFileSync(input);
}
const obj = JSON.parse(input);

Reply:
Thanks, had forgotten that it still needed to handle string inputs. Corrected.